scripts/update/readarr.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
**All** update functions run every time a `box update` is ran, this should be moved to the upgrade folder

Reply:
It also looks like none of these changes will be necessary on a fresh install, I can tell you more about the structure of the folders later